CONTRIBUTE.md

```diff
@@ -0,0 +1,42 @@
+<h1 align="center"><strong> Contribution Guideline</strong> </h1>
+
+<br>
+<h2><strong>How do I make a contribution?</strong></h2>
+
+<p>Never made an open source contribution before? Wondering how contributions work in the in our project? Here's a quick rundown!<p>
+<br>
+<h2> <strong> Here's a quick rundown on how to make a contribution to our project: </strong> </h2>
+<br>
+<ol>
+    <li> Find an <strong>issue</strong> that you are interested in addressing or a <strong>feature</strong> that you would like to add.</li>
+    <br>
+    <li>Fork the repository associated with the issue to your local GitHub organization. This means that you will have a copy of the repository under <strong>your-GitHub-username/repository-name. </strong> </li>
+    <br>
+    <li>Clone the repository to your local machine using: <strong> git clone https://github.com/github-username/repository-name.git. </strong></li>
```

Comment:
Steps that include some sort of code snippet or git command you can use markdown syntax. It will help developers to easily copy paste the command. 
For E.g 
```
git clone repository-link
```
You can refer this cheatsheet to get yourself familarize with markdown syntax [Markdown cheatsheet](https://www.markdownguide.org/cheat-sheet/)